tcms/testplans/tests/test_admin.py

```diff
@@ -0,0 +1,23 @@
+from django.urls import reverse
+from tcms.tests import LoggedInTestCase
+from tcms.utils.permissions import initiate_user_with_default_setups
+from tcms.tests.factories import TestPlanFactory
+
+
+class TestTestPlanAdmin(LoggedInTestCase):
+
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+        initiate_user_with_default_setups(cls.tester)
+        cls.test_plan = TestPlanFactory()
+
+    def test_regular_user_can_not_add_testplan(self):
```

Comment:
Naming! It doesn't matter what level of permissions the user has, nobody can access the Add page. Same for change.

Reply:
Why did you fix this and then undo it ?